source/fundamentals/builders/projections.txt

```diff
@@ -11,3 +11,169 @@ Projections Builders
    :class: singlecol
 
 .. _projections-builders:
+
+Overview
+--------
+
+MongoDB supports **field projection**, specifying which fields to include and exclude when returning results from a
+query. Projection in MongoDB follows some basic rules:
+
+- the :guilabel:`_id` field is **always** included and must explicitly be excluded
```

Comment:
Since these bullets aren't completing a sentence, I think they should be capitalized.